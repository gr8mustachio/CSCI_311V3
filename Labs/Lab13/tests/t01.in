11
addEdge 0 2
addEdge 0 3
addEdge 1 3
addEdge 1 4
addEdge 2 5
addEdge 4 10
addEdge 5 6
addEdge 5 8
addEdge 6 7
addEdge 6 8
addEdge 7 2
addEdge 9 4
addEdge 10 9
printGraph
sameCycle 5 7
sameCycle 4 6
sameCycle 0 2
sameCycle 0 7
sameCycle 10 4
sameCycle 2 8
