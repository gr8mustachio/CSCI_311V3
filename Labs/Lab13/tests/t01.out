2 3 
3 4 
5 

10 
6 8 
7 8 
2 

4 
9 
5 and 7 are on the same cycle.
4 and 6 are not on the same cycle.
0 and 2 are not on the same cycle.
0 and 7 are not on the same cycle.
10 and 4 are on the same cycle.
2 and 8 are not on the same cycle.
